### PR TITLE
Update to gtk4-rs 0.8 / cairo-rs 0.19

### PR DESCRIPTION
**Pull request by sdroege**
Depends on https://github.com/plotters-rs/plotters-cairo/pull/32

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -7,12 +7,14 @@ edition = "2018"
...
-git = "https://github.com/plotters-rs/plotters"
+version = "0.3.5"
 default_features = false
 features = ["surface_series"]
 
 [dependencies.plotters-cairo]
-git = "https://github.com/plotters-rs/plotters-cairo"
+version = "0.6.0"
+git = "https://github.com/sdroege/plotters-cairo"
+branch = "cairo-rs-0.19"
```
Comment (sdroege) [inline Cargo.toml]: This should be removed after merging the other PR.

Comment (AaronErhardt) [reply Cargo.toml]: The PR is merged and the new version is published on crates.io :)

Comment (sdroege) [reply Cargo.toml]: And updated to use the version from crates.io. Thanks!

**Review by AaronErhardt — APPROVED** [review]
Thanks!

**Pull request merged**